quoteHome.styles&markup

HliebVeklenko opened the pull request:
додано стилі до quote-home, трішки змінено розмітку в quote-home.html, додано картинки(6). Наче весь quote-home всьо(є питання з марджинами між секціями але на зустрічі запитаю)

Commit:
done

додано стилі до quote-home, трішки змінено розмітку в quote-home.html, додано картинки(6). Наче весь quote-home всьо

Commit:
Update quote-home.html

прибрав статичний текст, котрий потрібно змінювати з js (бо забув як дурник)

Commit:
Update quote-home.html

Hellwood1 commented on src/partials/quote-home.html:
```diff
@@ -4,7 +4,7 @@
         </svg>
         <h2 class="quotes-home-title">Quote of the day</h2>
-        <svg class="icon-inverted-commas">В
+        <svg class="icon-inverted-commas">
```
Чудово, що прибрав помилку!

Hellwood1 commented on src/partials/quote-home.html:
```diff
@@ -4,13 +4,11 @@
             <use href="../images/sprite.svg#icon-runing-man"></use>
```
До html претензій немає!

Commit:
Update quote-home.css

Hellwood1 committed:
Upd. padding for section

- Видалив margin-bottom та додав padding-top/bottom

Hellwood1 committed:
fix.

- Видалив папку img та додав картинки з секції hero в папку images
- Змінив шлях до картинок

Hellwood1 committed:
Merge branch 'main' into quoteHome.styles
